fe/fe-core/src/main/java/org/apache/doris/load/ExportTaskExecutor.java

```diff
@@ -89,16 +89,14 @@ public void execute() throws JobException {
         LOG.debug("[Export Task] taskId: {} updating state to EXPORTING", taskId);
         exportJob.updateExportJobState(ExportJobState.EXPORTING, taskId, null, null, null);
         List<OutfileInfo> outfileInfoList = Lists.newArrayList();
-        for (int idx = 0; idx < selectStmtLists.size(); ++idx) {
-            LOG.debug("[Export Task] taskId: {} processing statement {}/{}",
-                    taskId, idx + 1, selectStmtLists.size());
+        if (selectStmt.isPresent()) {
```

Comment:
In what situation the selectStmt can be null?

Reply:
If we export table without data, we would construct an `ExportTaskExecuto`r with empty `selectStmt`